src/server/auth/server/testing/admin_test.go

```diff
@@ -111,8 +95,11 @@ func TestActivate(t *testing.T) {
 	// Check that the token 'c' received from pachd authenticates them as "pach:root"
 	who, err := rootClient.WhoAmI(rootClient.Ctx(), &auth.WhoAmIRequest{})
 	require.NoError(t, err)
-	require.Equal(t, auth.ClusterRole_SUPER, who.ClusterRoles.Roles[0])
 	require.Equal(t, auth.RootUser, who.Username)
+
+	bindings, err := rootClient.GetClusterRoleBindings()
+	require.NoError(t, err)
+	require.Equal(t, buildClusterBindings(), bindings)
```

Comment:
Most of the changes in these tests are just 1:1 swaps of "check an ACL/check a cluster role binding" -> "check a role binding".